src/dbus_api/pool/mod.rs

```diff
@@ -87,7 +88,9 @@ where
                 .add_p(pool_3_0::has_cache_property(&f))
                 .add_p(pool_3_0::alloc_size_property(&f))
                 .add_p(pool_3_0::used_size_property(&f))
-                .add_p(pool_3_0::total_size_property(&f)),
+                .add_p(pool_3_0::total_size_property(&f))
+                .add_p(pool_3_0::total_size_property(&f))
```

Comment:
These two lines are identical, so one is redundant. I'm not sure what was intended here.

Reply:
This was a rebase issue.